src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf

```diff
@@ -442,6 +442,14 @@
                 <source>This URL is missing a top-level domain.</source>
                 <target>Cette URL doit contenir un domaine de premier niveau.</target>
             </trans-unit>
+            <trans-unit id="114">
+                <source>This value is too short. It should contain at least {{ min }} words.</source>
+                <target>Cette valeur est trop courte. Elle doit contenir au moins {{ min }} mots.</target>
```

Comment:
Perhaps picky here, just passing by

What if {{ min }} is 1 ?

How others core translations handle singular plural if any?

Reply:
That's right, I updated this PR and the other one to pluralize the message